packages/client/src/context/ThemeContext.tsx

```diff
@@ -0,0 +1,45 @@
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import { createContext, useMemo, useState } from 'react';
+
+import { darkTheme, defaultTheme } from '../theme';
+
+const ThemeContext = createContext({
+  toggleColorMode: () => {},
+});
+
+const ThemeContextProvider = (props: { children: JSX.Element[] | JSX.Element }) => {
+  const { children } = props;
```

Comment:
```suggestion
const ThemeContextProvider = ({ children }: { children: JSX.Element[] | JSX.Element }) => {

```

Można zdekonstruować od razu w parametrze.